MDANSE/Src/MDANSE/Framework/Jobs/IJob.py

```diff
@@ -276,12 +276,11 @@ def process_tasks_queue(self, tasks, outputs, log_queues):
         return True
 
     def _run_multicore(self):
+        if hasattr(self._status, "_queue_0"):
+            self._status._queue_0.put("started")
 
-        ctx = multiprocessing.get_context("spawn")
-
-        manager = ctx.Manager()
-        inputQueue = manager.Queue()
-        outputQueue = manager.Queue()
+        inputQueue = Queue()
+        outputQueue = Queue()
```

Comment:
I guess the manager bit was used because qsize doesn't work with non-manager queues on macos. However, the original version seems to cause a leaked process when terminate is called from the GUI.

The new version doesn't leak processes with terminate from the GUI but it means that a few parts of the code needed to be updated so that it avoided the call to qsize.